src/App.js

```diff
@@ -19,33 +19,35 @@ enableScreens()
 const App = (_props, _context) => {
   const [appState, setAppState] = useState<?string>(AppState.currentState)
 
-  const handleAppStateChange: (?string) => void = (nextAppState) => {
-    const previousAppState = appState
-
-    setAppState(nextAppState)
-    if (previousAppState != null && nextAppState === 'active') {
-      RNBootSplash.hide()
-    } else if (
-      previousAppState === 'active' &&
-      nextAppState != null &&
-      nextAppState.match(/inactive|background/)
-    ) {
-      RNBootSplash.show()
-    }
-  }
-
   // note: previously this was hanlded in the applicationDidEnterBackground
   // event in AppDelegate.m, but after moving to a .storyboard launch screen
   // that solution didn't seem feasible anymore.
-  if (Platform.OS === 'ios') {
-    useEffect(() => {
+  // eslint-disable-next-line consistent-return
```

Comment:
I would avoid removing rules (otherwise what's the point on adding them?). I think we can just return an empty clean up function in the other case.